jcommune-view/jcommune-web-controller/src/test/java/org/jtalks/jcommune/web/controller/PrivateMessageControllerTest.java

```diff
@@ -437,6 +411,20 @@ private PrivateMessage getPrivateMessage() {
                 new JCUser("username2", "email2", "password2"), "title", "body");
     }
 
+    /**
+     * @param hasGlobalError true if we have GlobalError like an ObjectError.
+     * @param errorFields list of fields witch have validation errors can be
+     *                    empty if we don't have any.
+     * @return MockMvc instance for unit testing.
+     */
+    private MockMvc getMockMvcWithValidator(boolean hasGlobalError, String... errorFields){
+        Validator validator = new ValidatorStub(errorFields);
+        if (hasGlobalError){
```

Comment:
Do I understand right that you're _setting_ a global error if the parameter is passed? This looks weird.. In this kind of tests we don't want to mock anything, we want to run them in a way similar to the real world. Therefore if you need error, you just create an invalid request.